hack/release.toml

```diff
@@ -6,98 +6,6 @@ github_repo = "siderolabs/omni"
 match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 pre_release = false
-previous = "v1.3.0"
+previous = "v1.4.0"
 
-[notes.aaa-urgent-sqlite-migration]
```

Comment:
I would probably keep this for visibility (so anyone hitting "latest" release still sees it)